src/BuildScriptGeneratorCli/Commands/BuildScriptCommand.cs

```diff
@@ -61,7 +67,18 @@ internal override int Execute(IServiceProvider serviceProvider, IConsole console
                 return ProcessConstants.ExitFailure;
             }
 
-            console.WriteLine(generatedScript);
+            if (string.IsNullOrEmpty(OutputPath))
+            {
+                console.WriteLine(generatedScript);
+            }
+            else
+            {
+                File.WriteAllText(OutputPath, generatedScript);
```

Comment:
Can you check if this api still works if a user supplies a path which does not have all the directories in the path? if not, we should create it IMO

Reply:
Added an extensions class for file system operations that includes a method for this.